projects/v2/src/app/static/url.ts

```diff
@@ -8,6 +8,15 @@ export function getAssetsURL(dataVersion, currentSheet) {
   return `assets/data/${dataVersion}/${currentSheet.name}.csv`;
 }
 
-export function getInformation(id: string, group = 1) {
-  return `https://www.ebi.ac.uk/ols/api/ontologies/uberon/terms?iri=http://purl.obolibrary.org/obo/${id.replace(':', '_')}`;
+export function buildASCTApiUrl(id: string, group = 1) {
+  return `/ebi_api/findByIdAndIsDefiningOntology?obo_id=${id}`;
```

Comment:
Try direct references for the build preview to see if CORS will ding us for these. If so, we can discuss next steps to address the issue.